### PR TITLE
code check test

### PR DESCRIPTION
**Pull request by aoimotoya**

### CODE REVIEW

File: badCode.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def badName
```
Comment (nagizerodev) [inline badCode.rb]: Naming/FileName: The name of this source file (`badCode.rb`) should use snake_case.

Comment (nagizerodev) [inline badCode.rb]: Naming/MethodName: Use snake_case for method names.

File: badCode.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+  if something
```
Comment (nagizerodev) [inline badCode.rb]: Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression.

Comment (nagizerodev) [inline badCode.rb]: Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.

File: badCode.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+    test
+    end
```
Comment (nagizerodev) [inline badCode.rb]: Layout/EndAlignment: `end` at 4, 4 is not aligned with `if` at 2, 2.

**Comment by nagizero** [comment]
hoge

File: bad_code.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+def badName
```
Comment (nagizerodev) [inline bad_code.rb:1]: Naming/MethodName: Use snake_case for method names.

File: bad_code.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+  if something
```
Comment (nagizerodev) [inline bad_code.rb:2]: Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression.

Comment (nagizerodev) [inline bad_code.rb:2]: Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.

**Pull request closed without merging**